lib/rubocop/cop/style/percent_literal_delimiters.rb

```diff
@@ -57,16 +60,28 @@ def on_percent_literal(node)
           add_offense(node, :expression)
         end
 
-        def preferred_delimiters(type)
-          cop_config['PreferredDelimiters'][type].split(//)
+        def preferred_delimiters
+          @preferred_delimiters ||=
+            if cop_config['PreferredDelimiters'].key?('default')
+              Hash[%w(% %i %I %q %Q %r %s %w %W %x).map do |type|
```

Comment:
And we can probably add some error checking for unknown delimiters in case users mistype something.